### DISCUSSION TITLE
Performance benchmarks / upper limits

### DISCUSSION THREAD

**Discussion by johncarter-** [ryangjchandler/orbit#112] [General]
I'm currently migrating an ecommerce site from Statamic to Filament. One of the reasons for moving from Statamic is because I'm hitting some bottlenecks in terms of performance.

I love flat-file data for all the obvious reasons, so I want to use Orbit. But I just wanted to know if you've got any benchmarks or known upper limits on how much data this bad boy can handle.

My store isn't huge - it has ~13,000 individual variations of ~1400 products. Does that sound like it will be a problem? What are the factors to consider?

**Comment by ryangjchandler** [comment]
@johncarter What's the relationship, does a Product have many Variations?

**Reply by johncarter-** [reply]
Yep that's it.

**Reply by ryangjchandler** [reply]
Yeah, so right now the SQLite cache gets invalidated when ANY file gets changed. For large datasets, that's less than perfect because data is likely changing quite often. 

I do have an issue open to improve this though, which I'll be working on this week likely (vacating atm).

**Reply by johncarter-** [reply]
Yeah no stress enjoy the break.

I will probably run some benchmarks myself, and when I do I can share the results here.
